README.md

```diff
@@ -71,6 +71,7 @@ Installation instructions for previous releases can be found in [Metrics Server
 
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
+0.7.x          | `metrics.k8s.io/v1beta1`  | 1.19+
```

Comment:
There are two other areas where the version is hardcoded, would you mind updating them as well? 
```sh
README.md:70:0.6.x          | `metrics.k8s.io/v1beta1`  | 1.19+
README.md:144:docker run --rm registry.k8s.io/metrics-server/metrics-server:v0.6.0 --help
manifests/components/release/kustomization.yaml:6:  newTag: v0.6.0
```

Reply:
Done